package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "codeclimate",
+      "description": "The path to the `codeclimate` executable."
+    },
+    "init": {
```

Comment:
You might want to give this a `"title"`, otherwise Atom will just show it as "Init".